SlevomatCodingStandard/Sniffs/Commenting/ForbiddenCommentsSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Commenting;
+
+use SlevomatCodingStandard\Helpers\DocCommentHelper;
+
+class ForbiddenCommentsSniff implements \PHP_CodeSniffer\Sniffs\Sniff
+{
+
+	public const CODE_COMMENT_FORBIDDEN = 'CommentForbidden';
+
+	/** @var string[] */
+	public $forbiddenCommentPatterns = [];
+
+	/**
+	 * @return mixed[]
+	 */
+	public function register(): array
+	{
+		return [
+			T_DOC_COMMENT_OPEN_TAG,
+		];
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
+	 * @param \PHP_CodeSniffer\Files\File $file
+	 * @param int $pointer
+	 */
+	public function process(\PHP_CodeSniffer\Files\File $file, $pointer): void
+	{
+		$comments = DocCommentHelper::getDocCommentDescription($file, $pointer);
+
+		if ($comments === null) {
+			return;
+		}
+
+		foreach ($this->forbiddenCommentPatterns as $forbiddenCommentPattern) {
```

Comment:
Please check that pattern contains delimiters.

Reply:
It would be nice to use normalized patterns with `SniffSettingsHelper::normalizeArray()` so it's possible to write comma in XML after last pattern.

Reply:
Both done.